site/content/docs/reference/cli.md

```diff
@@ -43,6 +43,9 @@ pinniped get kubeconfig [flags]
 - `--concierge-authenticator-type string`:
 
   Concierge authenticator type (e.g., 'webhook', 'jwt') (default: autodiscover)
+- `--concierge-mode`:
+
+Concierge mode of operation (e.g. 'ImpersonationProxy', 'TokenCredentialRequestAPI')(default: TokenCredentialRequestAPI)
```

Comment:
Correct me if I am wrong, but I think I see some new impersonation proxy CLI flags that aren't in this doc (e.g., `--concierge-credential-issuer`). Should we add those?